shared/zsh/dash.plugin.zsh

```diff
@@ -0,0 +1,29 @@
+#compdef dash
+#autoload
+
+dash () {
+  $1.sh
+}
+
+_dash() {
+  local -a commands
+
+  commands=(
```

Comment:
Please add the recently merged `make-frontend.sh` command - https://github.com/ricardoasmarques/ceph-dev-docker/pull/36

Reply:
Done